src/main/webapp/pages/samples/sample_files.html

```diff
@@ -182,6 +182,58 @@ <h4 class="media-heading" th:text="#{samples.files.empty.title}"></h4>
 			</table>
 		</div>
 	</div>
+
+	<div class="row" th:if="${not #lists.isEmpty(fast5)}">
```

Comment:
If we have fast5 files in here it still shows the "No sequencing data" message.  This should be hidden if there are fast5 files.

![image](https://user-images.githubusercontent.com/8951708/80997358-cc7e5200-8e06-11ea-8fe5-6a388342460d.png)

Reply:
Updated in 04448d4847cb221c0c8cc7f3cffd2e242c5054ed